gaffer-as-a-service/gaas-common/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>uk.gov.gchq.gaffer</groupId>
+        <artifactId>gaffer-as-a-service</artifactId>
+        <version>1.13.0</version>
+    </parent>
+    <artifactId>gaas-common</artifactId>
+    <version>1.13.0</version>
+    <name>gaas-common</name>
+    <description>Gaas common</description>
+
+    <properties>
+        <kubernetes.version>11.0.0</kubernetes.version>
+        <slf4j.api.version>1.7.25</slf4j.api.version>
+        <junit.version>5.7.0</junit.version>
+        <apache.commons.text.version>1.9</apache.commons.text.version>
+        <compiler.plugin.version>2.3.2</compiler.plugin.version>
+        <surefire.version>3.0.0-M5</surefire.version>
```

Comment:
Some of the version properties can be moved to the parent pom

Reply:
Done